i18n/locales/japanese/translations.json

```
@@ -32,23 +32,35 @@
     "go-to-front-page": "フロントページに移動"
   },
   "social-row": {
-    "author-has-twitter": "お読みいただきありがとうございます。ぜひ著者にツイートしてお知らせください。{{ <0> }}著者宛てにツイート{{ </0> }}",
-    "author-no-twitter": "この記事が参考になりましたらぜひシェアしてください。{{ <0> }}ツイート{{ </0> }}",
-    "default-tweet": "この記事が参考になりました。{{ twitter }} さんありがとうございます。"
+    "cta": {
+      "tweet-a-thanks": "お読みいただきありがとうございます。ぜひ著者にツイートしてお知らせください。{{ <0> }}著者宛てにツイート{{ </0> }}",
+      "tweet-it": "この記事が参考になりましたらぜひシェアしてください。{{ <0> }}ツイート{{ </0> }}"
+    },
+    "tweets": {
+      "default": "この記事が参考になりました。{{ author }} さんありがとうございます。",
+      "translation": "この役立つ記事を書いてくれた{{ author }}と、翻訳してくれた{{ translator }}に感謝します。"
```

```suggestion
      "translation": "この役に立つ記事を書いてくれた {{ author }} さんと、翻訳してくれた {{ translator }} さんに感謝します。"
```

I added spaces around them to make sure the text don't interfere with the twitter handles.

---

Awesome, thanks for catching that :slightly_smiling_face: